src/controllers/list.js

```diff
@@ -610,7 +610,7 @@ define(["globalize", "listView", "layoutManager", "userSettings", "focusManager"
             }
 
             if (item) {
-                return item.Name;
+                return globalize.translate(item.Name);
```

Comment:
Are we sure this should be translated by the web client? It was unclear in chat exactly what's getting translated here.

Reply:
here item.Name is returning dynamic library name which needs to be translated. This can be removed if translation of dynamic library is not required.

Reply:
I have a non-blocking issue with this, that's really more of a maintenance issue.

It potentially introduces strings in translations that aren't anywhere in the code. As such, there's a high chance of strings being removed from translations at some point (since we use code as a reference for cleaning strings).

And what if the name returned has a space in it? Is that a case we need to account for?

I think having the API for libraries being able to translate names itself would be better, but that requires way more engineering.

I'm okay with merging this as-is, but I think we need to rethink this once we rework the APIs.